### DISCUSSION TITLE
Do you have enum support?

### DISCUSSION THREAD

**Discussion by Hoss3770** [dineug/erd-editor#273] [Ideas]
Thanks for the extension 

I did not find enum support 

Do you have enum support or plan to add it ? 

Thank you?

**Comment by dineug** [comment]
Enum has no plans at the moment to support it.

You can further define datatype.
Sadly, the table will be longer.

![89310249-66c48f00-d6af-11ea-8a1e-f03bab7f950e](https://user-images.githubusercontent.com/45829489/129528289-0007a398-08b2-484c-87f7-c78919a7f618.png)
